lib/rubocop/cop/rspec/predicate_matcher.rb

```diff
@@ -149,12 +149,23 @@ def check_explicit(node) # rubocop:disable Metrics/MethodLength
           return if part_of_ignored_node?(node)
 
           predicate_matcher?(node) do |actual, matcher|
+            next unless replaceable_matcher?(matcher)
+
             add_offense(node, message: message_explicit(matcher)) do |corrector|
               corrector_explicit(corrector, node, actual, matcher, matcher)
             end
           end
         end
 
+        def replaceable_matcher?(matcher)
+          case matcher.method_name.to_s
+          when 'include'
+            matcher.arguments.count <= 1
```

Comment:
We don't have a test case with zero arguments, do we?

Reply:
I thought the same thing. Also, which implementation of include takes zero arguments?

Reply:
```suggestion
            matcher.arguments.one?
```
May be just this if there are no cases where it's used with no arguments. And `include` does not accept a block, so `expect(foo).to include` is nonsense and is probably worth a cop if anyone have seen this in practice, or there are offenders in real-world apps open source code.